py-polars/polars/dataframe/frame.py

```diff
@@ -1116,6 +1116,55 @@ def _replace(self, column: str, new_column: Series) -> Self:
         self._df.replace(column, new_column._s)
         return self
 
+    @property
+    def plot(self) -> Any:
+        """
+        Create a plot namespace.
+
+        Polars does not implement plotting logic itself, but instead defers to
+        hvplot. Please see the `hvplot reference gallery <https://hvplot.holoviz.org/reference/index.html>`_
+        for more information and documentation.
+
+        Examples
+        --------
+        Scatter plot:
+
+        >>> df = pl.DataFrame(
+        ...     {
+        ...         "length": [1, 4, 6],
+        ...         "width": [4, 5, 6],
+        ...         "species": ["setosa", "setosa", "versicolor"],
+        ...     }
+        ... )
+        >>> df.plot.scatter(x="length", y="width", by="species")  # doctest: +SKIP
+
+        Line plot:
+
+        >>> from datetime import date
+        >>> df = pl.DataFrame(
+        ...     {
+        ...         "date": [date(2020, 1, 2), date(2020, 1, 3), date(2020, 1, 3)],
+        ...         "stock_1": [1, 4, 6],
+        ...         "stock_2": [1, 5, 2],
+        ...     }
+        ... )
+        >>> df.plot.line(x="date", y=["stock_1", "stock_2"])  # doctest: +SKIP
+
+        For more info on what you can pass, you can use ``hvplot.help``:
+
+        >>> import hvplot  # doctest: +SKIP
+        >>> hvplot.help("scatter")  # doctest: +SKIP
+        """
+        try:
+            import hvplot.polars  # noqa: F401
+        except ModuleNotFoundError:
+            raise ModuleNotFoundError(
+                "hvplot is not installed, or is older than version 0.9.0.\n\n"
+                "Please see https://hvplot.holoviz.org/getting_started/installation.html "
+                "for installation instructions."
+            ) from None
```

Comment:
Why not reraise this error?

```suggestion
        except ModuleNotFoundError as exc:
            raise ModuleNotFoundError(
                "hvplot is not installed, or is older than version 0.9.0.\n\n"
                "Please see https://hvplot.holoviz.org/getting_started/installation.html "
                "for installation instructions."
            ) from exc
```